### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BenHurCreations**
The new README file contains all the relevant information about our system's inital specification, and our focus while developing the MVP

### CODE REVIEW

**Review by simzacks — APPROVED** [review]

**Review by simzacks — COMMENTED** [review]
Any reason why you want to use Mongo instead of a relational database (mysql or postgresql)?
Note, in the classes we are only going to go into relational databases.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,60 @@
...
+
+________________________________
+
+## Initial System Spesification:
+
+**SuperX purpose:** prices comparison between supermarkets in Israel
+  
+#### User Story:
+1. User logs in to the website and chooses his relevant grocery
+2. The user enters "Prices Comparison" button
```
Comment (simzacks) [inline README.md:17]: He'll probably have to choose the products he wants as well.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,60 @@
...
+
+1. Python based script that sends request to the supermarkets server every defined period of time, and downloads the prices files from them.
+
+2. Python based parser that reads the xml prices files and sends them to the relevant spot in the DB
+
+#### Database:
+
+1. Contains system's data- products prices by firm, branch & date
+2. Saves users' copmared shopping carts' prices
+*MongoDB based
```
Comment (shaibrown5) [inline README.md]: I thought we were using mysql. the Db i built so far is a mysql DB not mongo

File: README.md (hunk 1 above)
Comment (shaibrown5) [reply README.md:17]: that is written in part 1 "chooses relevant grocery"

**Comment by shaibrown5** [comment]
sounds good all in all, there are some grammar mistakes, but that I can change afterwards.

**Comment by BenHurCreations** [comment]
changed & ready to be merged

**Pull request merged**